tests/integration-tests/tests/networking/test_cluster_networking/test_cluster_in_private_subnet/pcluster.config.ini

```diff
@@ -0,0 +1,23 @@
+[global]
+cluster_template = default
+
+[aws]
+aws_region_name = {{ region }}
+
+[cluster default]
+base_os = {{ os }}
+key_name = {{ key_name }}
+vpc_settings = parallelcluster-vpc
+scheduler = {{ scheduler }}
+master_instance_type = {{ instance }}
+compute_instance_type = {{ instance }}
+initial_queue_size = 1
+max_queue_size = 1
+maintain_initial_size = true
```

Comment:
let's set this to false so that in case of slurm we have the bootstrap logic submit a job to the scheduler (this happens only when initial_size > min_size ==> maintain_initial_size = false)